### ISSUE TITLE
Strategies for accepting CSV multiplex data in the UP

### ISSUE THREAD

**Issue by dylansmith-Nava** [CDCgov/prime-reportstream#12709]
### User Story
As a ReportStream teammate, I want to map out the internal ReportStream business logic and STLT business value for various strategies of accepting multiplex data from Senders, so that the engineering team can implement the chosen strategy when ready.

### Background
Owing to the [CSV Support in the UP MVP](https://docs.google.com/document/d/1dfvSWQjPhAKHW2shQ2jn4VxAPrLpvNJ4duK27NY0gnU/edit#heading=h.1pruvfgadxev) Product Brief, some 50% of current COVID pipeline Senders send messages in CSV format. As we transition these Senders to the UP, we are considering various strategies to both 'meet users where they are' while helping to support ReportStream values and key OKRs of allowing organizations to send multiple conditions to the UP.

**Option 1**: CSV multiplex data is sent to STLTs as individual messages 
- Each submitted multiplex result will be one row in the CSV file, and will be received by the STLT as separate HL7 messages or FHIR bundles
- Pros: Much easier to do, basically the same logic as any other single analyte condition
- Cons: We are not sure if STLTs will accept the data in this form as satisfactory

**Option 2**: Create a "unique identifier" data column in the CSV UP schema, and ask Senders to fill out this column upon submission for multiplex data
- Pros: Would allow a ReportStream engineer to link multiplex data together so it can be sent in one package
- Cons: We are not sure if Senders are capable of doing this option, and we will need to do research with Senders to understand their capabilities

### Risks/Impacts/Considerations
- We won't be sure if it would be unacceptable to any Receiver(s) in the future to receive multiplex results separately, and if not tackling this work right now is just postponing issues
- Linking multiplex results is difficult and would require engineering capacity that we don't have right now

### Open Questions
- Are separate multiplex results OK for STLTs?

### Links
- [CSV schema proposal in UP Product Brief](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7Bc9d75dbd-ed18-4682-927a-23f10e69e766%7D&action=edit&wdPid=7e299cf5)
- [CSV File Formats](https://docs.google.com/spreadsheets/d/1WumZYfUURgQWu1ihvK4NgUbR2ms0VE2MfzspDx22eTw/edit#gid=9297054)
- [12613](https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/12613): Business case and logic flow for multiple options of sending CSV messages to the UP

### Acceptance Criteria
- [x] Clear and conclusive strategy for accepting multiplex results that is good enough to be accepted by engineering team
- [ ] Alternative strategies to pursue for sending multiplex data to STLTs

### Out of Scope
- Time and scoping estimate for implementation

**Comment by dylansmith-Nava** [comment]
Queried the prime-design channel and a NIH stakeholder that @arnejduranovic recommended. Waiting on feedback and hopefully will set up a few meetings to learn more.

**Comment by dylansmith-Nava** [comment]
We learned from NIH stakeholder that at least for respiratory OTC multiplex data, jurisdictions prefer to receive each test result as separate. So for this type of multiplex test, we can treat the test virtually the same as single analyte data. See [12/29 conversation](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7B6E3B2196-1472-4CEF-8F17-4FCB83B58B46%7D&file=CSV%20in%20UP%20notes%20and%20synthesis.docx&action=default&mobileredirect=true) for reference. 

For all other mutiplex results, a customer support ticket has been generated on our behalf to be answered by the APHL team. This ticket has been significantly delayed. As such, any multiplex decisions have been delegated as "[Unfinished Work](https://cdc.sharepoint.com/:w:/r/teams/ReportStream/_layouts/15/Doc.aspx?sourcedoc=%7Bc9d75dbd-ed18-4682-927a-23f10e69e766%7D&action=edit&wdPid=5c491f69)" in the CSV schema product brief. I may be able to answer any other multiplex questions when this customer support ticket is returned, but I can make new ZenHub ticket(s) for that. As a result, @Andrey-Glazkv suggested closing this ticket out for now. CC @jillian-hammer
